### ISSUE TITLE
Story Page feedback 

### ISSUE THREAD

**Issue by duquennoisa** [CUCentralAdvancement/essential-cu#303]
Hi @alexfinnarn 
Not sure if this is the best place to do this so let me know if you'd like it somewhere else or on a different issue... 

1. Nav bar 
I'd like the nav to be sticky-- unless you think it shouldn't for any reason? 

![image](https://user-images.githubusercontent.com/70037329/135321166-3a35b998-3aa6-4db6-a063-b80dbe039f56.png)
- curve is smaller than design. Trying to match ECU branding and the curve is bigger. 
- no dropshadow between the nav bar layers. 

![image](https://user-images.githubusercontent.com/70037329/135323229-5ab0e2f2-bfc2-4ec8-b623-4370bceea68f.png)
- Buttons feel too large and button text is in all caps

2. Story hero area- 

![image](https://user-images.githubusercontent.com/70037329/135321817-79339ee2-3162-4d66-897a-90813c6fb8fc.png)

Story title font needs to be bigger and bolder 
sub-text also bigger and bolder. 
sharing icons are transparent filled rather than white and the grey is different and the feel too big. 

3. Body 

![image](https://user-images.githubusercontent.com/70037329/135322322-bab47c53-85dd-48d9-927f-90e8da81155c.png)
- Image should be larger than body for desktop. 
![image](https://user-images.githubusercontent.com/70037329/135322444-9a51f753-51f2-433a-ae20-8d697e55ce76.png)
-font should be bigger and bolder. 

4. Related funds
![image](https://user-images.githubusercontent.com/70037329/135322970-867503a8-df01-4f60-abb8-b0200ba4cc0c.png)

Underline and italicize... I want to make it look like a link.

5. Cards 
We talked yesterday about keeping the size static and making them wrap based on break points. 
![image](https://user-images.githubusercontent.com/70037329/135322873-1df017c9-54f2-4b36-9657-eb8c416216ce.png)
Campus name should be black 

![image](https://user-images.githubusercontent.com/70037329/135322671-170d5e31-a86b-4229-a80f-50a628048757.png)
We can remove this since we aren't connecting to a story bank. 

Ill keep adding but thats my first pass

**Comment by alexfinnarn** [comment]
@duquennoisa I've updated the "Story One" page with a lot of the suggestions made except mainly sizing ones, e.g. font-size, border-radius, bold. I'll check Sketch inspector to see...but if you have an idea of what the base font-size, line-height, font-family...etc...is then...

Actually, we have what was used last year if you want the same bases.

As far as the format for feedback, the most helpful way for me is via checklist linked to the explanation. This way, people can see the progress at the top of the issue with what's changed and keep updating the items and explanation areas as things are updated. I'll post an example below.

---

My brief issue intro...

Design Changes:
- [ ] [Nav should be sticky](#sticky)
- [ ] [Font size off](#font-size)

<div id="sticky"></div>
## Sticky Nav 

Here is my problem and screenshot...

<div id="font-size"></div>
## Font size

...another issue...

**Comment by Feissit** [comment]
For this QA I viewed both pages at 100% and tried to provide side by side screenshots of the issue. I tried to avoid issues that Anne brought up already.

- [ ] [Scare quote](###Scare quote)
- [ ] [Spacing between buttons] (###Button spacing)
- [ ] [Social media icons](###Social media)
- [ ] [Hero image](###Hero image)
- [ ] [Body text margins](###Body text margins)
- [ ] [Body image width](###Body image width)
- [ ] [Appreciate icons](###Appreciate icons)
- [ ] [Arrows on links](###Arrows on links) 
- [ ] [Card edges](###Card edges)
- [ ] [Subtitle text](###Subtitle text)
- [ ] [Campus tags](###Campus tags)
- [ ] [Give now buttons](###Give now buttons)
- [ ] [Footer spacing/alignment](###Footer)



###Scare quote
Scare quote line widths are different from mockup, extending too far on each side, text width is also different. Quotation mark styles are different
![Screen Shot 2021-09-30 at 8 54 36 AM](https://user-images.githubusercontent.com/91230278/135479383-d4419252-34f2-4d8a-b297-5f3177869b7e.png)
![Screen Shot 2021-09-30 at 8 54 31 AM](https://user-images.githubusercontent.com/91230278/135479500-fef665f0-abec-471b-a7d8-e513da344160.png)

###Button spacing
Spacing between buttons is wider in mockup. External page Icon on "giving at CU" differ.
![Screen Shot 2021-09-30 at 8 57 13 AM](https://user-images.githubusercontent.com/91230278/135479982-56203acf-feb8-47bf-8969-2a8b5b9bd51c.png)

###Social media
Social media icons differ, they do appear to be smaller on the mockup.
![Screen Shot 2021-09-30 at 9 00 33 AM](https://user-images.githubusercontent.com/91230278/135480546-835ea11c-72cb-4fff-8d16-9d7e875f8cc3.png)

###Hero image
Hero image size and where title/intro text starts differs 
![Screen Shot 2021-09-30 at 9 04 18 AM](https://user-images.githubusercontent.com/91230278/135481095-b2c10a56-9eb1-4d8f-b701-f89ee7554532.png)

###Body text margins
Margins are different widths in body text
![Screen Shot 2021-09-30 at 9 05 06 AM](https://user-images.githubusercontent.com/91230278/135481297-90198a10-f703-4936-8244-62891e128926.png)

###Body image width
Body image width differ from mockup
![Screen Shot 2021-09-30 at 9 06 04 AM](https://user-images.githubusercontent.com/91230278/135481492-2ca91fd7-5546-4962-9777-0fc6374ecdc2.png)

###Appreciate icons
Appreciate icons are different. They're not filled in on the mockup, style is different on thumbs up icon and happy face icon.
![Screen Shot 2021-09-30 at 9 09 20 AM](https://user-images.githubusercontent.com/91230278/135482040-cb88c05d-06d3-4892-8167-db7d99365687.png)

###Arrows on links
Arrows are next to the link in mockup, at the far right of the link module in Heroku
![Screen Shot 2021-09-30 at 9 12 57 AM](https://user-images.githubusercontent.com/91230278/135482755-d2fcdef9-830b-4c38-92d6-16ab298f0f01.png)

###Card edges
Top of the card edges are rounded, and cards start with a margin on each side of the page.
![Screen Shot 2021-09-30 at 9 15 12 AM](https://user-images.githubusercontent.com/91230278/135483021-97b2a0b9-c461-41b2-8a5e-d3364823a331.png)

###Subtitle text
Subtitle text on cards is lighter in mockup
![Screen Shot 2021-09-30 at 9 16 29 AM](https://user-images.githubusercontent.com/91230278/135483317-b9b8ee89-b68b-41d3-9a8a-9a233bf2726b.png)

###Campus tags
Campus tags have slightly more square edges in mockup
![Screen Shot 2021-09-30 at 9 17 03 AM](https://user-images.githubusercontent.com/91230278/135483466-3bb671ff-14fa-40ae-b02b-98176fdf9784.png)

###Give now buttons
###Footer
- [ ] Give now buttons do not match. Squarer in Heroku, and all caps. Text is too big. I'm assuming the spacing/alignment of links etc will be addressed at a later point?
![Screen Shot 2021-09-30 at 9 18 16 AM](https://user-images.githubusercontent.com/91230278/135483734-fb3444a2-8c55-4f25-8c1e-d82ba6dd0172.png)

**Comment by alexfinnarn** [comment]
@duquennoisa and @Feissit thanks for the review and input. I did update the code for most issues, but certainly, there are still differences between design and what you see on the current website.

As mentioned in some comments, the margins/paddings/font sizes aren't exactly the same, but with modern, responsive web design the values tend to be ratios and not exact measurements so people usually try to get ratios right instead of values. 

I did try "viewport units" for fonts, but that didn't work out at all. To explain a bit more...the user's screen equals 100 viewpoint units in bot height and width. If I say the font size is `5vh` for five height units, then it would equate to 5% of the total screen size. That may look good on mobile but not on tablet or desktop. 

So, I went back to using pixels in the code and matching what InVision gives me. However, I swear some text that says it is `#000000`, or black, looks to be some shade of gray. 

I will close this issue now and open some targeted issues so the conversation can focus on one problem at a time.
